polus-montage-plugin/src/filepattern.py

```diff
@@ -0,0 +1,439 @@
+import re, copy
```

Comment:
Can we load this file from the utils folder? Either as part of multi-stage build where you copy this file in, or via git.

Reply:
That's a good idea. Actually, a better option would probably be to do a pip install pointed at the utils folder similar to what I do with bfio.

Reply:
I can do this in future plugins, or once the `filepattern` util is merged. Otherwise merging this will merge in the `filepattern` util.